dozer-sql/expression/src/wasm/udf.rs

```diff
@@ -0,0 +1,98 @@
+use super::error::Error::{WasmFunctionMissing, WasmTrap};
+use crate::error::Error::{self, Wasm};
+use dozer_types::ordered_float::OrderedFloat;
+use dozer_types::types::{Field, Record, Schema};
+
+use wasmtime::*;
+
+use crate::execution::Expression;
+
+pub fn evaluate_wasm_udf(
+    schema: &Schema,
+    name: &str,
+    config: &str,
+    args: &[Expression],
+    record: &Record,
+) -> Result<Field, Error> {
+    let input_values = args
+        .iter()
+        .map(|arg| arg.evaluate(record, schema))
+        .collect::<Result<Vec<_>, Error>>()?;
+
+    let engine = Engine::default();
+    let module = Module::from_file(&engine, config).unwrap();
+    let mut store = Store::new(&engine, ());
+    let instance = Instance::new(&mut store, &module, &[]).unwrap();
+
+    let wasm_udf_func;
+    match instance.get_func(&mut store, name) {
+        Some(func) => {
+            wasm_udf_func = func;
+        }
+        None => {
+            return Err(Wasm(WasmFunctionMissing(
+                name.to_string(),
+                config.to_string(),
+            )));
+        }
+    }
```

Comment:
These code should go into the compilation phase? Just like the onnx UDF is passing in a `Session` instead of the model path.